dateutil/test/test_parser.py

```diff
@@ -579,6 +579,11 @@ def testFuzzyIgnoreAMPM(self):
 
         self.assertEqual(parse(s1, fuzzy=True), datetime(1945, 1, 29, 14, 45))
 
+    def testFuzzyWithDotNotation(self):
+        s1 = "27.08"
+        today_year = date.today().year
+        self.assertEqual(parse(s1, fuzzy=True), datetime(today_year, 8, 27, 0, 0))
```

Comment:
This should be changed in the following ways:

1. Just pass a specific date to `default` so you don't have to worry about retrieving the current year
2. Remove the `fuzzy` keyword. It is unnecessary.